Packs/CortexResponseAndRemediation/Playbooks/playbook-Exchange_forwarding_rule_configured.yml

```diff
@@ -1,25 +1,7 @@
-  Containment:\n\n- The playbook checks if the IP or domain of the forwarding email
-  address is malicious. If so, it suggests blocking the IP using PAN-OS while continuing
-  the investigation in parallel.\n\nInvestigation:\n\n- The playbook checks for suspicious
-  behaviors, including whether an Exchange admin created the rule outside of working
-  hours, from unusual geolocation, or if the user who created the rule has a high-risk
-  score. It then aggregates all evidence collected during the investigation.\n\nContainment:\n\n-
-  If at least two suspicious pieces of evidence are found, the playbook executes soft
-  response actions, including signing the user out and deleting the forwarding email
-  address from the user account mailbox. The user will be notified of these actions
-  via email.\n- If more than two pieces of suspicious evidence are found, the playbook
-  will initiate hard response actions. These include disabling the user and removing
-  the forwarding email address from their mailbox. The user will be notified of these
-  actions via email.\n\nRequirements: \n\nFor any response action, you need the following
-  integrations:\n- EWS Extension Online Powershell v3 integration.\n- Azure Active
-  Directory Users."
+description: "This playbook addresses the following alerts:\n\n- External Exchange inbox forwarding rule configured.\n- Suspicious Exchange inbox forwarding rule configured.\n- Suspicious Exchange email-hiding inbox rule.\n- Possible BEC Exchange email-hiding inbox rule.\n- Exchange email-hiding transport rule based on message keywords.\n- Suspicious Exchange email-hiding transport rule.\n- Exchange transport forwarding rule configured.\n- Suspicious Exchange transport forwarding rule configured.\n\nPlaybook Stages:\n \nTriage: \n\n- The playbook retrieves the caller's IP, the forwarding email address, and the domain.\n\nEarly Containment:\n\n- The playbook checks if the IP or domain of the forwarding email address is malicious. If so, it suggests blocking the IP using PAN-OS while continuing the investigation in parallel.\n\nInvestigation:\n\n- The playbook checks for suspicious behaviors, including whether an Exchange admin created the rule outside of working hours, from unusual geolocation, or if the user who created the rule has a high-risk score. It then aggregates all evidence collected during the investigation.\n\nContainment:\n\n- If at least two suspicious pieces of evidence are found, the playbook executes soft response actions, including signing the user out and disabling the forwarding rule from the user account mailbox. \n- If more than two pieces of suspicious evidence are found, the playbook will initiate hard response actions. These include disabling the user and the forwarding rule from the user account mailbox. \n\nRequirements: \n\nFor any response action, you need the following integrations:\n- EWS Extension Online Powershell v3 integration.\n- Azure Active Directory Users."
```

Comment:
I would add to the containment plan the fact that the user disable action is an analyst decision,